<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArticleDisplayerController</class>
 <widget class="QWidget" name="ArticleDisplayerController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="firstPageBtn">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../MindReader/res/resource.qrc">
       <normaloff>:/img/first</normaloff>:/img/first</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="prevPageBtn">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../MindReader/res/resource.qrc">
       <normaloff>:/img/backward</normaloff>:/img/backward</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="ADCIndex"/>
   </item>
   <item>
    <widget class="QPushButton" name="nextPageBtn">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../MindReader/res/resource.qrc">
       <normaloff>:/img/forward</normaloff>:/img/forward</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="lastPageBtn">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../MindReader/res/resource.qrc">
       <normaloff>:/img/last</normaloff>:/img/last</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="page">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../MindReader/res/resource.qrc">
       <normaloff>:/img/page</normaloff>:/img/page</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">ADCPageFormat</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pageBreak">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../MindReader/res/resource.qrc">
       <normaloff>:/img/page-break</normaloff>:/img/page-break</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">ADCPageFormat</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>50%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>150%</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200%</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zoomInBtn">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../MindReader/res/resource.qrc">
       <normaloff>:/img/zoomIn</normaloff>:/img/zoomIn</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSlider">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zoomOutBtn">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../MindReader/res/resource.qrc">
       <normaloff>:/img/zoomOut</normaloff>:/img/zoomOut</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../MindReader/res/resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="ADCPageFormat"/>
 </buttongroups>
</ui>
